docs/components/modeler/web-modeler/git-sync.md

```diff
@@ -189,6 +189,17 @@ To perform hotfixes or patches of production or production-bound processes, sync
 Creating multiple copies of a process application can complicate navigation and deployment if you have multiple files with the same ID in a project. To avoid this, you can create copies of the process application in different projects.
 :::
 
+## Self-Managed environment variables
+
+You can set the following environment variables to configure Git sync in a Self-Managed environment:
+
+| Provider      | Variable                                     | Description                                                                                                                   | Default                     |
+| ------------- | -------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------- | --------------------------- |
+| All providers | `CAMUNDA_MODELER_GITSYNC_MAX_FILES`          | Maximum number of allowed files for sync operations.                                                                                    | `50`                        |
```

Comment:
I think the name should be without underscore (see also https://docs.spring.io/spring-boot/reference/features/external-config.html#features.external-config.typesafe-configuration-properties.relaxed-binding). I've tried both variants and only the one without underscore worked for me (in similar cases in the past Spring accepted both if I remember correctly 🤷🏻).
```suggestion
| All providers | `CAMUNDA_MODELER_GITSYNC_MAXFILES`          | Maximum number of allowed files for sync operations.                                                                                    | `50`                        |
```